packages/zosjobs/__tests__/__unit__/MonitorJobs.unit.test.ts

```diff
@@ -614,5 +655,368 @@ describe("MonitorJobs", () => {
                 expect(error.message).toMatchSnapshot();
             });
         });
+        describe("initial status check", () => {
+            it("should return immediately if the initial status is OUTPUT", async () => {
+
+                // Mock GetJobs.getStatusCommon
+                const mockedGetJobs = jest.fn(async (args): Promise<any> => {
+                    return {jobname: "FAKE", jobid: "FAKE", status: "OUTPUT"};
+                });
+                GetJobs.getStatusCommon = mockedGetJobs;
+
+                // IJob requires more fields to be populated - fake it out with only the required here.
+                const iJobParms: any = {jobname: "FAKE", jobid: "FAKE"};
+                const response = await MonitorJobs.waitForJobOutputStatus(new Session({hostname: "fake", port: 443}), iJobParms);
+                expect(response).toMatchSnapshot();
+                expect(mockedGetJobs).toHaveBeenCalledTimes(1);
+            });
+
+            it("should return immediately if the initial status is the expected status", async () => {
```

Comment:
Is this the same as the test above?

Reply:
yes, ive merged the new test title to be: "should return immediately if the initial status is the expected status, ie: OUTPUT/ OUTPUT"